.travis.yml

```diff
@@ -21,16 +21,53 @@ after_success:
 
 jobs:
   include:
-    - stage: integration
-      scala: 2.13.1
+
+    - stage: integration-20.x
+      scala: 2.13.3
+      env: AKKA_TEST_TIMEFACTOR=1.5 AKKA_TEST_LOGLEVEL=OFF ES_TEST_ADDRESS_PORT=1114 ES_TEST_HTTP_PORT=2114 ES_TEST_IS_20_SERIES=true
+      before_install:
+        - docker pull quay.io/ahjohannessen/eventstore-20.6.0-bionic
```

Comment:
@ahjohannessen are we missing an official image?

Reply:
@t3hnar Just for a while, when they sync with docker hub I’ll change to that :)